po/es.po

```diff
@@ -20513,6 +20518,16 @@ msgstr "Atajos del teclado"
 msgid "Action"
 msgstr "Acción"
 
+#~ msgid "date time"
+#~ msgstr "fecha/hora"
+
+#~ msgid "select images to be geotagged"
+#~ msgstr "Seleccione imágenes para ser geoetiquetadas"
+
+#, fuzzy
+#~ msgid "apply GPX file"
+#~ msgstr "adjuntar archivo gpx de recorrido..."
```

Comment:
Same comment, GPX instead of gpx.

Note: Really this one was the trigger to check for other occurrences.